garageScript/c0d3-app
# Update DropdownMenu to set selected item as active one

Closes #2188 
Related to #1975

# Problem

When an item is selected, it doesn't get set as the active item.

# Solution

Create a state and set the selected item as the active one.

# Test

1. Click the dropdown menu
2. Select an item
3. Notice how it gets set as the dropdown menu text next to the arrow

---

[Storybook](https://deploy-preview-2189--youthful-thompson-448332.netlify.app/?path=/story/components-dropdownmenu--basic)
